calc/package.scala

```diff
@@ -0,0 +1,81 @@
+package distcompiler.calc
+
+import cats.syntax.all.given
+
+import scala.collection.mutable
+
+import distcompiler.*
+import dsl.*
+import distcompiler.calc.tokens.*
+
+object tokens:
+  object Expression extends Token:
+    override def showSource: Boolean = true
```

Comment:
this one might actually be better = false, because it doesn't have underlying text, strictly speaking